### PR TITLE
Fix texts

### PR DESCRIPTION
**Pull request by weblaetitia**

### CODE REVIEW

File: public/actions/decryptDataInKms.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { BACKEND_URL } from "./backendConfig";
 //
 // Decrypt data in KMS
 //
-export const decryptDataInKms = async (encryptText: Uint8Array, kmsToken: string, decryptionKey: string): Promise<string> => {
+export const decryptDataInKms = async (encryptText: Uint8Array, kmsToken: string, decryptionKeyID: string): Promise<string> => {
```
Comment (ccorsin) [inline public/actions/decryptDataInKms.ts:7]: a detail - but `decryptionKeyUid` to stay coherent with above ?

File: public/actions/encryptDataInKms.ts [hunk 2]
```diff
@@ -4,9 +4,14 @@ import { BACKEND_URL } from "./backendConfig";
 //
 // Encrypt data in KMS
 //
-export const encryptDataInKms = async (clearData: string, kmsToken: string, accessPolicy: string, mpkID: string): Promise<Uint8Array> => {
+export const encryptDataInKms = async (
+  clearData: string,
+  kmsToken: string,
+  accessPolicy: string,
+  masterPublikKeyId: string
```
Comment (ccorsin) [inline public/actions/encryptDataInKms.ts:11]: typo - masterPublikKeyId -> masterPublicKeyId

General comment as above : keep only one syntax ? xxxxUid vs. xxxxID

File: src/CoverCrypt.tsx [hunk 3]
```diff
@@ -383,13 +364,18 @@ const CoverCrypt: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
 
       <Stack spacing={3}>
         <Text fontSize="md">
-          Consider the following 2 policy axes, Department and Country which data are partitioned by the following attributes:
+          Consider 2 policy axes: <Text as="b">Department</Text> and <Text as="b">Country</Text>. Each axes are partitioned by attributes:{" "}
+          <Text as="b">Marketing and HR</Text> for the Departement axe and <Text as="b">France, Spain and Germany</Text> for the Country
```
Comment (ccorsin) [inline src/CoverCrypt.tsx:368]: typo - <Text as="b">Marketing and HR</Text> for the Departement **axis** and <Text as="b">France, Spain and Germany</Text> for the Country **axis**

File: src/LoginPage.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const LoginPage: React.FC<LoginPageProps> = ({ loginWithRedirect }) => {
+  return (
+    <Box w="100%" h="100vh" display="flex" alignItems="center" justifyContent="center" bgGradient="linear-gradient(to-br,#220c54,#300193)">
+      <Box
+        w="100%"
+        h="100vh"
+        display="flex"
+        alignItems="center"
+        justifyContent="space-between"
+        // backgroundImage={CosmianEncryption}
```
Comment (ccorsin) [inline src/LoginPage.tsx:16]: remove this?

File: src/PKI.tsx [hunk 5]
```diff
@@ -313,14 +317,13 @@ const PKI: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
           )}
 
           {/* 3 - CLIENT 2: PUBLISH WRAPPED Public KEY in KMS */}
-          <HeadingWithCode heading="Publish the wrapped Public Key" code="/src/actions/uploadPemInPKI.ts" />
+          <HeadingWithCode heading="Publish certificate" code="/src/actions/uploadPemInPKI.ts" />
           <CodeHighlighter codeInput={code?.uploadPemInPKI} />
           <Text>
-            <ClientBadge client={2}>Client 2</ClientBadge> publishes its public key <Code>pk_2</Code> wrapped with the certificat in the{" "}
-            <Text as="b">SaaS KMS</Text>
+            <ClientTwo /> publishes its public key <Code>pk_2</Code> wrapped as a certificate in the <Text as="b">SaaS KMS</Text>
```
Comment (ccorsin) [inline src/PKI.tsx:323]: wrapped **in** a certificate ?

File: src/PKI.tsx [hunk 6]
```diff
@@ -375,7 +379,7 @@ const PKI: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
           <HeadingWithCode heading="Send wrapped Decryption Key in Saas KMS" code="/src/actions/uploadKeyInPKI.ts" />
           <CodeHighlighter codeInput={code?.uploadKeyInPKI} />
           <Text>
-            <ClientBadge client={1}>Client 1</ClientBadge> send wrapped Decryption Key in <Text as="b">SaaS KMS</Text>.
+            <ClientOne /> send wrapped Decryption Key in <Text as="b">SaaS KMS</Text>.
```
Comment (ccorsin) [inline src/PKI.tsx:382]: sends ?

File: public/actions/encryptDataInKms.ts (hunk 2 above)
Comment (weblaetitia) [reply public/actions/encryptDataInKms.ts:11]: xxxxUid

**Pull request merged**